python/src/aica_api/client.py

```diff
@@ -37,10 +37,14 @@ def __init__(self, url: str = 'localhost', port: Union[str, int] = '5000'):
         elif '//' in url or ':' in url:
             raise ValueError(f'Invalid URL format {url}')
         else:
-            self._address = f'http://{url}:{port}'
+            if port == 5000:
```

Comment:
We should make the default port `8080` no?

Reply:
like this ? 507e0a7fcb51c45466b9c7c352c6cdf9724cba56

Reply:
Yes, perfect.